### ISSUE TITLE
NavigationDrawer component example code at https://smeltejs.com/components/navigation-drawers seems to be out of date

### ISSUE THREAD

**Issue by kuhlaid** [matyunya/smelte#93]
The sample code https://smeltejs.com/components/navigation-drawers does not seem compatible with Smelte version 1.0.11. From what I can tell the property names have changed or were removed, such as 'showDesktop'. Is that correct?

**Comment by hoyeungw** [comment]
Got the same problem

**Comment by ppusapati** [comment]
Yes, When trying to implement as it is in example it gives an error

'default' is not exported by node_modules/smelte/src/index.js, imported by src/routes/drawer/Drawer.svelte
